tests/integration/test_dbapi_integration.py

```diff
@@ -532,6 +532,15 @@ def test_null_date_with_time_zone(trino_connection):
     assert rows[0][0] is None
 
 
+def test_binary_query_param(trino_connection):
+    cur = trino_connection.cursor(experimental_python_types=True)
+
+    cur.execute("SELECT ?", params=(bytearray("a", "utf-8"),))
```

Comment:
Also add a case with non-utf-8 bytes (to make sure implementation isn't doing the stupid thing of converting to `str` and then decoding/encoding to byte-array).